user-guide/web-report-designer.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Web Report Designer
+page_title: Web Report Designer
+description: Web Report Designer
+slug: web-report-designer
+tags: report,designer
+published: true
+position: 200
+---
+
+# Web Report Designer
+
+Since R2 2019, we offer a web-based report designer which allows creating and designing reports in the Report Server. 
+Further information about this controll can be found in [Telerik Web Report Designer article](https://docs.telerik.com/reporting/web-report-designer).
```

Comment:
Shorten the link, e.g. '../web-report-designer'. The same is valid for all links to 'docs.telerik.com'